sample_reefperf_config/sample_app_deployment.json

```diff
@@ -0,0 +1,23 @@
+{
+	"app-deployment": {
+		"nodes": [
+			{
+				"name": "cache",
+				"type": "cache",
+				// additional args auth_data that allows login to cloud node (exec locally)
```

Comment:
there are no comments in valid json. Also, this is not a good place for documentation.